website/docs/cli/commands/modules.mdx

```diff
@@ -17,15 +17,13 @@ key, source, and version.
 
 ## Usage
 
-Usage: `terraform modules -json`
+Usage: `terraform modules [options]`
 
-The following flags are available:
+The following optional flags are available:
 
 - `-json` - Displays the module declarations in a machine-readable, JSON format.
 
-The `-json` flag is _required_ to run the `terraform modules` command. In future releases, we will extend this command to allow for additional options.
-
-The output of `terraform modules` includes a `format_version` key, which is set to the value of `"1.0"` in Terraform 1.10.0. The semantics of this version are:
+The output of `terraform modules -json` includes a `format_version` key, which is set to the value of `"1.0"` in Terraform 1.10.0. The semantics of this version are:
 
 - For minor versions, such as `"1.1"`, changes or additions will be backward-compatible.
   Ignore object properties that are unrecognized to remain forward-compatible
```

Comment:
Sorry, opportunistic nit here, but on the line below could you replace `e.g.` with "such as"? We don't use latin abbreviations in the docs.

Reply:
Included 👍 

Also apologies, we ended up going with a different design that led to some changes to the associated docs. My mistake in setting the PR as ready to review too soon!

Reply:
Oh no worries! Just let me know whenever you'd like a review, then!